io.github.btpf.alexandria.yml

```diff
@@ -0,0 +1,56 @@
+---
+id: io.github.btpf.alexandria
+command: "alexandria"
+rename-icon: "alexandria"
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+# Use rust-nightly to get once_cell working on older rust runtime compatible with org.gnome.platform 42
+# https://github.com/tauri-apps/tauri/discussions/4426#discussioncomment-4838945
+# flatpak install runtime/org.freedesktop.Sdk.Extension.rust-nightly/x86_64/21.08
+- org.freedesktop.Sdk.Extension.rust-nightly
+- org.freedesktop.Sdk.Extension.node16
+separate-locales: false
+finish-args:
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--share=ipc"
+- "--share=network"
+- "--device=dri"
+- "--filesystem=/run/media"
+modules:
+- name: alexandria
+  buildsystem: simple
+  build-options:
+    append-path: /usr/lib/sdk/node16/bin:/usr/lib/sdk/rust-nightly/bin
+    env:
+      CARGO_HOME: "/run/build/alexandria/cargo"
+      npm_config_nodedir: /usr/lib/sdk/node16
+      npm_config_offline: 'true'
+  build-commands:
+  # Builds libmobi-rs dependency. Will attempt to run unit test which will fail in flatpak environment
+  - cd ./libmobi-rs && sh build.sh
+  # Fetch packages
+  - npm ci --offline --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache
+  - cargo --offline fetch --manifest-path src-tauri/Cargo.toml
+  # Build the tauri deb with the opt_once_cell feature enable to get once_cell working on older rust verison
```

Comment:
why not using the once_cell crate?

Reply:
I will take your advice on this. once_cell is a part of std, and I have a workaround to just enable it in nightly for the flatpak. I was thinking a workaround is better than the dependency. I am mostly indifferent to both solutions, I just forgot that once_cell was once it's own crate.